### PR TITLE
osmox

### PR DESCRIPTION
**Pull request by Hussein-Mahfouz**
Related to #19 and #39. We can either:
- implement logic in this [comment](https://github.com/Urban-Analytics-Technology-Platform/acbm/issues/19#issuecomment-2077533236)
- point the user to osmox and tell them where to place the output file once they've created it

### CODE REVIEW

**Comment by sgreenbury** [comment]
Two tasks:
- [ ] Update the output format to geoparquet
- [ ] Update the output path to `interim/osmox` and change the downstream scripts to point to this

**Comment by sgreenbury** [comment]
Adding the deps mentioned for [osmium](https://pypi.org/project/osmium/) fixes the CI for Python 3.12 ([b54cff0](https://github.com/Urban-Analytics-Technology-Platform/acbm/pull/40/commits/b54cff08f46c03e84237ddef2aec2c9c7f850d25))

**Comment by Hussein-Mahfouz** [comment]
Do these dependencies need to be added to the poetry lock file?

**Comment by sgreenbury** [comment]
> Do these dependencies need to be added to the poetry lock file?

Since they're external system deps to python I don't think we're able to install with poetry though we can add this to a requirements section in the docs and link to the [osmium docs](https://pypi.org/project/osmium/)

**Comment by sgreenbury** [comment]
Merging this now with (the output file from pyrosm can be passed directly to osmox ([1504c59](https://github.com/Urban-Analytics-Technology-Platform/acbm/pull/40/commits/1504c5975b2feb9c24ea973139399565006332c5)).

File: osmox/osmox_notes.txt
Comment (sgreenbury) [inline osmox/osmox_notes.txt:1]: Perhaps we can move these osmox notes to a section in the documentation as part of #63?

Comment (Hussein-Mahfouz) [reply osmox/osmox_notes.txt:1]: Does it make sense to add the notes as a subsection under the [Installation](https://github.com/Urban-Analytics-Technology-Platform/acbm/tree/62-documentation-for-repo?tab=readme-ov-file#installation) section of the main README? If that sounds good, do you mind doing it? I think you're better placed to add the notes than I am.

Comment (sgreenbury) [reply osmox/osmox_notes.txt:1]: Sounds good, I'll go ahead and add these notes to the docs as part of #63. I think we can have a section on configuration of the pipeline and we can include a subsection on osmox and mention the default config file JSON (`config_osmox.json`) along with the descriptions here.

**Pull request merged**